docs/developer/plugin-list.asciidoc

```diff
@@ -485,7 +485,7 @@ using the CURL scripts in the scripts folder.
 |{kib-repo}blob/{branch}/x-pack/plugins/metrics_entities/README.md[metricsEntities]
 |This is the metrics and entities plugin where you add can add transforms for your project
 and group those transforms into modules. You can also re-use existing transforms in your
-modules as well.
+newly modules as well.
```

Comment:
`new modules` or `newly created modules`?

Reply:
I will fix it to say `newly created modules`